client/src/components/ProfileDetail/RequestCard.tsx

```diff
@@ -0,0 +1,85 @@
+import { format } from 'date-fns';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import InputLabel from '@material-ui/core/InputLabel';
+import { Rating } from '@material-ui/lab';
+
+import useStyles from './useStyles';
+
+interface Props {
+  profile: {
+    priceRate: number;
+  };
+}
+export default function RequestCard({ profile }: Props): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Box textAlign="center">
+        <Typography variant="h4">{`${profile.priceRate}/hr`}</Typography>
+        <Rating value={4.2} />
+      </Box>
+      <Grid container>
+        <Grid item xs={12}>
+          <InputLabel htmlFor="dropIn" className={classes.inputLabel}>
+            Drop In
+          </InputLabel>
+        </Grid>
+        <Grid item sm={12} xs={12} className={classes.inputFieldsContainer}>
+          <TextField
+            type="date"
+            id="dropIn"
+            name="dropIn"
+            variant="outlined"
+            inputProps={{
+              min: format(new Date(), 'yyyy-MM-dd'),
+            }}
+            // sadas
+          />
+          <TextField
+            type="time"
+            id="dropInTime"
+            name="dropInTime"
+            variant="outlined"
+            // sadas
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <InputLabel htmlFor="dropOff" className={classes.inputLabel}>
+            Drop Off
+          </InputLabel>
+        </Grid>
+        <Grid item sm={12} xs={12} className={classes.inputFieldsContainer}>
```

Comment:
You could just do `xs={12}` and it would span 12 columns from xs-xl

Reply:
Oops, I forgot to remove this, Thanks